File: src/Data Structure/Linked List/Creating A Node Dobly Linked List.cpp
#include <bits/stdc++.h>
using namespace std;

typedef struct node Node;

struct node{
    int data;
    Node *next;
    Node *prev;
};

Node *create_node (int item, Node *next, Node *prev)
{
    Node *new_node = (Node *) malloc(sizeof(Node));
    if (new_node==NULL) {
        printf("Error! Could not create a new node\n");
        exit(1);
    }
    new_node->data = item;
    new_node->next = next;
    new_node->prev = prev;
    return new_node;
}

int main()
{
    return 0;
}

File: src/Data_Structure/Linked_List/Creating_A_Node_Dobly_Linked_List.java
package Data_Structure.Linked_List;

public class Creating_A_Node_Dobly_Linked_List {
  static class Node {
    int data;
    Node next;
    Node prev;
  }

  static Node createNode(int item, Node next, Node prev) {
    Node newNode = new Node();

    newNode.data = item;
    newNode.next = next;
    newNode.prev = prev;

    return newNode;
  }

  public static void main(String[] args) {
    Node doblyLinkedList = createNode(10, null, null);
  }
}
